File: StraviaTec-Web/Models/Carrera.cs
using System;
using System.Collections.Generic;

namespace StraviaTec_Web.Models
{
    public partial class Carrera
    {
        public Carrera()
        {
            CategoriaCarrera = new HashSet<CategoriaCarrera>();
            CuentaBancaria = new HashSet<CuentaBancaria>();
        }

        public int Id { get; set; }
        public int IdRecorrido { get; set; }
        public int IdEvento { get; set; }
        public string Nombre { get; set; }
        public DateTime Fecha { get; set; }
        public decimal Costo { get; set; }

        public virtual Evento IdEventoNavigation { get; set; }
        public virtual Recorrido IdRecorridoNavigation { get; set; }
        public virtual ICollection<CategoriaCarrera> CategoriaCarrera { get; set; }
        public virtual ICollection<CuentaBancaria> CuentaBancaria { get; set; }
    }
}

File: StraviaTec-Web/ClientApp/src/app/models/carrera.ts
export class Carrera {
    nombre: string;
    actividad: number;
    fecha: string;
    cuentaBancaria: string;
    categoria: number;
    patrocinador: number;
    costo: number;    
    recorrido: string;

    constructor(
        Nombre: string,
        Actividad: number,
        Fecha: string,
        CuentaBancaria: string,
        Categoria: number,
        Patrocinador: number,
        Costo: number,
        Recorrido: string
    ){
        this.nombre = Nombre;
        this.actividad = Actividad;
        this.fecha = Fecha;
        this.cuentaBancaria = CuentaBancaria;
        this.categoria = Categoria;
        this.patrocinador = Patrocinador;
        this.costo = Costo;
        this.recorrido = Recorrido;
    }
}